cmd/mr/scripts/test-mr.sh

```diff
@@ -199,7 +199,14 @@ timeout -k 2s 180s ../../worker/mrworker ../../apps/early_exit/early_exit.so &
 # `jobs` ensures that any completed old processes from other tests
 # are not waited upon
 jobs &> /dev/null
-wait -n
+
+sum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
+newSum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
```

Comment:
Rename sum and newSum to initial_proc_count and proc_count

Directly initialise proc_count with initial_proc_count